openedx/adg/common/lib/zoom_client/tests/test_client.py

```diff
@@ -0,0 +1,279 @@
+    (WEBINAR_NOT_FOUND_RESPONSE, False),
+], ids=[
+    'webinar-panelists-remove-successful',
+    'invalid-user',
+    'webinar-not-found'
+], indirect=['api_response'])
+@pytest.mark.django_db
+def test_remove_panelists(api_response, expected_result, webinar_with_zoom_data, zoom_config, mocker):
+    """
+    Test that the panelists are removed successfully
+    """
+    _, _, zoom_api_response = api_response
+
+    mocked_zoom_delete_request = mocker.patch.object(_ZoomClient, 'delete_request')
+    mocked_zoom_delete_request.return_value = zoom_api_response
+
+    actual_result = AdgZoomClient().remove_panelists(webinar_with_zoom_data.id)
+
+    mocked_zoom_delete_request.assert_called_with(f'/webinars/{webinar_with_zoom_data.id}/panelists')
+    assert actual_result == expected_result
+
+
+# pylint: disable=protected-access
+@pytest.mark.parametrize('api_response, expected_result', [
+    (ZOOM_GET_USER_SUCCESSFUL_RESPONSE, True),
+    (USER_DOES_NOT_EXIST_RESPONSE, False),
+    (INVALID_USER_RESPONSE, False),
+], ids=[
+    'user-with-valid-account',
+    'user-not-found',
+    'user-with-invalid-permissions'
+], indirect=['api_response'])
+def test_get_account_information(api_response, expected_result, zoom_config, mocker):
+    """
+    Test that the `get_account_information` returns the account information correctly
+    """
+    _, content, zoom_api_response = api_response
+    mocked_zoom_users_api = mocker.patch.object(_ZoomClient, 'user')
+    mocked_zoom_users_api.get.return_value = zoom_api_response
+
+    account_information = AdgZoomClient()._get_account_information()
+
+    if expected_result:
+        assert 'account_number' in account_information
+        assert content['account_number'] == account_information['account_number']
+
+        assert 'account_id' in account_information
+        assert content['account_id'] == account_information['account_id']
+    else:
+        assert not account_information
+
+
+@pytest.mark.parametrize('api_response, expected_result', [
+    (
+        (
+            HTTP_200_OK,
+            {
+                'type': AdgZoomClient.LICENSED_ACCOUNT_TYPE,
+                'account_id': DUMMY_ACCOUNT_ID,
+                'account_number': DUMMY_ACCOUNT_NUMBER
+            }
+        ),
+        True
+    ),
+    (
+        (
+            HTTP_200_OK,
+            {
+                'type': AdgZoomClient.LICENSED_ACCOUNT_TYPE,
+                'account_id': DUMMY_ACCOUNT_ID - 1,
+                'account_number': DUMMY_ACCOUNT_NUMBER
+            }
+        ),
+
+        False
+    ),
+    (
+        (
+            HTTP_200_OK,
+            {
+                'type': AdgZoomClient.LICENSED_ACCOUNT_TYPE,
+                'account_id': DUMMY_ACCOUNT_ID,
+                'account_number': DUMMY_ACCOUNT_NUMBER - 1
+            }
+        ),
+        False
+    ),
+    (
+        (HTTP_200_OK, {'type': 1, 'account_id': DUMMY_ACCOUNT_ID, 'account_number': DUMMY_ACCOUNT_NUMBER}),
```

Comment:
Let's format this and break it into multiple lines as we have done for the above tuples for consistency.